### PR TITLE
Bench TPS tweaks for transaction fees

### PR DESCRIPTION
**Pull request by pgarg66**
#### Problem
The bench TPS client does not account for transaction fees while initially funding the accounts.

#### Summary of Changes
Set aside lamports for fee while funding accounts.

#4530

### CODE REVIEW

**Comment by pgarg66** [comment]
https://github.com/solana-labs/solana/pull/4527

File: bench-tps/src/bench.rs [hunk 1]
```diff
@@ -25,8 +25,9 @@ use std::thread::Builder;
 use std::time::Duration;
 use std::time::Instant;
 
-pub const MAX_SPENDS_PER_TX: usize = 4;
-pub const NUM_LAMPORTS_PER_ACCOUNT: u64 = 20;
+pub const MAX_SPENDS_PER_TX: u64 = 4;
+pub const NUM_LAMPORTS_PER_ACCOUNT: u64 = 128;
+pub const NUM_LAMPORTS_PER_TX: u64 = 1;
```
Comment (mvines) [inline bench-tps/src/bench.rs]: Why not fetch this from the cluster when we get a blockhash

Comment (pgarg66) [reply bench-tps/src/bench.rs]: sounds good. I was wondering how to get this value during runtime..

Comment (pgarg66) [reply bench-tps/src/bench.rs]: Done

**Comment by sagar-solana** [comment]
This doesn't address the "do_bench_tps" problem right?

**Comment by pgarg66** [comment]
@mvines , is this good to go?

**Pull request merged**